### PR TITLE
Added new profession: Muckraker

### PR DESCRIPTION
**Pull request by Luggggsmi**
#### Summary
Content "Added Warjournalist Profession"

#### Testing
1.- Made a new character.
2.- Played for an hour.

#### Additional context

Beulah Killian _2024-03-14T00-59-32

### CODE REVIEW

**Comment by XygenSS** [comment]
- I don't think war corrsspondants are supposed to be armed
- Trapping skill is a bit odd?
- Generally the profession is pretty out of place considering New England was not a warzone

**Comment by DoctorBoomstick** [comment]
Generally, I’d like it if this were renamed to a basic reporter/journalist profession; there’s no need to pick one style of correspondent over another. I’m neutral over the gun, because, well, New England was being invaded by a lovecraft’s assortment of monsters, so I wouldn’t think it out of the norm that somebody packed along a carry gun. The combat shirt is a bit of a strange inclusion; I don’t know if there’s any basis for that. Also, the inclusion of:

"traits": ["LIAR,"  "LIGHTSTEP"]

Provoked a mile-wide eyebrow rase.

**Comment by AnotherSeawhite** [comment]
What is happpening in this PR???

**Comment by cdru123** [comment]
> What is happpening in this PR???

It's a reference to Fear & Hunger 2: Termina, basically

**Comment by Luggggsmi** [comment]
Yeah, tbh i didn't think this through too much, i just want to test some stuff, my bad.

**Comment by Luggggsmi** [comment]
And what's wrong with the traits? Did I miss something there?

**Comment by DoctorBoomstick** [comment]
> And what's wrong with the traits? Did I miss something there?

I just found it a tad strange that we’re implying that every reporter is great at lying and soft-footing about.

**Comment by Luggggsmi** [comment]
Subterfuge is a useful skill, when information gathering is necessary, lol.

File: data/json/professions.json [hunk 1]
```diff
@@ -7273,6 +7273,27 @@
       "female": { "entries": [ { "item": "panties" }, { "item": "bra" } ] }
     }
   },
+  {
+    "type": "profession",
+    "id": "muckracker",
+    "name": "Muckracker",
+    "description": "You've covered it all before, you know? War, Corruption, the bizzare and more, but even a pulitzer couldn't prepare you for this scoop.",
```
Comment (TheShadowFerret) [inline data/json/professions.json:7280]:
```suggestion
    "description": "You've covered it all before, you know? War, corruption, the bizarre and more, but even a Pulitzer couldn't prepare you for this scoop.",
```
I've covered typos you know.
dead-rising-frank-west

File: data/json/professions.json [hunk 2]
```diff
@@ -7273,6 +7273,27 @@
...
+    "proficiencies": [ "prof_lockpicking" ],
+    "points": 3,
+    "skills": [ { "level": 2, "name": "pistol" }, { "level": 2, "name": "speech" } ],
+    "items": {
+      "both": {
+        "items": [ "jacket_leather", "dress_shirt", "dress_shoes", "tie_necktie" ],
```
Comment (TheShadowFerret) [inline data/json/professions.json:7286]: No press ballistic vest?

Comment (Luggggsmi) [reply data/json/professions.json:7286]: i don't know, i imagine a muckraker to be more low-down, you know? Not trying to get much attention to themselves.

**Comment by cdru123** [comment]
I think they could benefit from having their Speech skill raised to 4 (same as the photojournalist), or perhaps even higher to better correspond to the reference. As for gear, they should have a smartphone (like the majority of middle-class professions), and likely a spare magazine for their gun (since civilians who carry guns will usually also carry a spare reload).

**Comment by Zireael07** [comment]
Why close? I liked the idea (after name change)

**Pull request closed without merging**